drivers/i3c/Kconfig.renesas

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config I3C_RENESAS_MUX_PRIORITY
+	int "I3C Renesas Mux Init Priority"
```

Comment:
Both `*_PRIORITY` summary need to be clarified since they are obviously different kconfigs.

Reply:
Good point, let me do that

Reply:
They are different Kconfigs, but one is useless without the other. 
I hope you will find my new comments explain the situation much better. Thank you!